pages/u/[username]/index.tsx

```diff
@@ -124,6 +123,11 @@ export const getServerSideProps = async (context: UserSSRPropsContext) => {
   const userData = (await req.json()) as DbUser;
   const ogImage = `${process.env.NEXT_PUBLIC_OPENGRAPH_URL}/users/${username}`;
 
+  // Cache page for 60 seconds
+  context.res.setHeader("Cache-Control", "public, max-age=0, must-revalidate");
+  context.res.setHeader("Netlify-CDN-Cache-Control", "public, max-age=0, stale-while-revalidate=60");
+  context.res.setHeader("Cache-Tag", `user-profiles,user-profile-${username}`);
```

Comment:
I added cache tags here so in a follow up PR, we could increase the cache time and also purge the cache key when someone updates their user profile.